### ISSUE TITLE
bug(localization): Locales out of date

### ISSUE THREAD

**Issue by edcarroll** [edcarroll/ng2-semantic-ui#223]
With #217 being merged, a number of locales are out of date (minor change needed).

@maksim-m, @ciaraward, @gotenxds, @fniwes, please can you update them accordingly? 😄

**Comment by edcarroll** [comment]
With #214 merged, locales now have support for `timesOfDay`, `timesOfDayUpperCase` and `timesOfDayLowercase`. This defines the AM / PM values. Using `aa` in the date formats is also now fully supported.

Please could you also add these values to the definitions? Thanks! 🎉

**Comment by edcarroll** [comment]
@mcosta74 are the Italian translations still up to date with the new `timesOfDay` e.t.c? @genuinefafa could you also please check this?

**Comment by genuinefafa** [comment]
Spanish doesn't use am/pm notation, si we need to set it up anyways? I mean, you might, but it's not common notation. (Weird tho, that Microsoft Windows localization for Spanish doea use am/pm, weird indeed). 

If required, copying from en-gb does the trick, but might be confusing for someone only opening that file. 🤔

**Comment by edcarroll** [comment]
I think it would be good to have the entire set of localisations, to at least show that they aren't missing if you see what I mean?

The Dutch locale has a little comment that just says it doesn't use them but hey - perhaps copy that?

**Comment by genuinefafa** [comment]
Yep. Copying will do the trick. But my point is that it isn't necessary but to make things more understandable. 

Will add those lines in a few days and do a PR. ☺️
